docs/howto_act_on_failure.md

```diff
@@ -0,0 +1,375 @@
+# Handling Vertex AI Pipelines Failures
+
+In Vertex AI Pipelines, failures can occur during the execution of a pipeline. To handle these failures, you can use Exit Handlers.
+
+## Exit Handlers
+
+Official documentation for Exit Handlers is available [here](https://www.kubeflow.org/docs/components/pipelines/v2/pipelines/control-flow/#exit-handling-dslexithandler).
+
+Exit Handlers are special steps that are executed whenever a pipeline exits, regardless of whether the pipeline’s steps have completed successfully or have failed.
+You can use them to  send notifications, or perform any other action that you want to occur when a pipeline exits with a certain status (e.g. failure).
+
+Here is an example of how to use Exit Handlers:
+
+```python
+import kfp.dsl
+
+@kfp.dsl.pipeline(name="my-pipeline")
+def my_pipeline():
+
+    my_exit_task = ...  # Define your exit task here. (instanciated component)
+
+    with kfp.dsl.ExitHandler(my_exit_task, name="Exit Handler"):
+        # Add your pipeline tasks here.
+```
+
+## Sending Notifications
+
+### Email notification using `VertexNotificationEmailOp`
+
+Official documentation for this component is available [here](https://cloud.google.com/vertex-ai/docs/pipelines/email-notifications).
+
+Vertex AI Pipelines provides a built-in component to send email notifications.
+This component is called `VertexNotificationEmailOp`.
+It is available in the `google_cloud_pipeline_components` package.
+You can use it as an exit handler in your pipeline to send email notifications when the pipeline exits.
+
+```python
+import kfp.dsl
+from google_cloud_pipeline_components.v1.vertex_notification_email import VertexNotificationEmailOp
+
+@kfp.dsl.pipeline(name="my-pipeline")
+def my_pipeline():
+
+    notification_email_task = VertexNotificationEmailOp(recipients=["john.doe@foo-bar.com"])
+
+    with kfp.dsl.ExitHandler(notification_email_task, name="Exit Handler"):
+        # Add your pipeline tasks here.
+```
+
+### Slack notification using custom component
+
+You can also use a custom component to send notifications.
+Here is an example of a custom component that sends a notification to a Slack channel when a pipeline job ends.
+
+!!! success "Prerequisites"
```

Comment:
this doesn't render properly for me: 

<img width="708" alt="image" src="https://github.com/artefactory/vertex-pipeline-starter-kit/assets/18250390/caac1e57-264a-4fb2-b946-98c81b897091">

Reply:
Do you use tech docs plugin to render it ? It's not pure markdown